polaris/polaris/sep31/info.py

```diff
@@ -0,0 +1,92 @@
+from typing import Dict
+
+from django.utils.translation import gettext as _
+from rest_framework.decorators import api_view, renderer_classes
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.renderers import JSONRenderer, BrowsableAPIRenderer
+
+from polaris.models import Asset
+from polaris.utils import render_error_response, Logger
+from polaris.integrations import registered_send_integration
+
+
+logger = Logger(__name__)
+
+
+@api_view(["GET"])
+@renderer_classes([JSONRenderer, BrowsableAPIRenderer])
+def info(request: Request) -> Response:
+    info_data = {
+        "receive": {},
+    }
+    for asset in Asset.objects.filter(sep31_enabled=True):
+        try:
+            fields = registered_send_integration.info(asset, request.GET.get("lang"))
+        except ValueError:
+            return render_error_response("unsupported 'lang'", content_type="text/html")
```

Comment:
This is the integration function, so if the developer using Polaris throws an error from `info()` it will be caught here. Polaris assumes the error is due not being able to support the language, since there is no other reason the `/info` endpoint would ever return a 400 response.